docs/datastore/alloydb.md

```diff
@@ -194,11 +194,11 @@ Private IP.
     ```bash
     CREATE EXTENSION vector;
     ```
-
-1. In a separate shell terminal, change into the root of the project:
+    
+1. Change into the service directory:
 
     ```bash
-    cd database-query-extension
+    cd database-query-extension/extension_service
```

Comment:
Can we separate out this fix into it's own PR? I know it's a small thing, but if we roll back for some reason we want it to stick around